docs/api/types.md

```diff
@@ -114,7 +114,10 @@ schema({
 
 ## `enum`
 
-Creates an enum type based on a TypeScript `enum` structure.
+With `enum` you can either :
```

Comment:
These docs files are generated from the JSDoc in the `src/` files. Please edit those original files first, then run `yarn docs` to generate these ones.